lib/constants/borderStyle.js

```diff
@@ -1,21 +1,23 @@
+/* eslint-disable */
```

Comment:
We should not disable eslint completely, if necessary - disable specific rules. Same for other files below.

Reply:
@givehug could you please as well fix this.